charts/workflows/charts/move2kube/templates/01-sonataflow_m2k.yaml

```diff
@@ -255,11 +254,15 @@ spec:
   podTemplate:
     container:
       resources: {}
-      image: quay.io/orchestrator/serverless-workflow-move2kube:4fae4bd85a48f4091f5c67a83fbcc94dcd29f1f2
+      image: {{ .Values.workflow.image }}
+      #imagePullPolicy: Always
```

Comment:
pls remove the comment.

Reply:
you see the "draft" in the status? :)

But better safe than sorry, I'll remove it once I get it working ;)

Reply:
so that was just a draft comment from my side :-P